script/dev

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# This is a script for running Compose inside a Docker container. It's handy for
+# development.
+#
+# $ ln -s `pwd`/script/dev /usr/local/bin/docker-compose
+# $ cd /a/compose/project
+# $ docker-compose up
+# 
+
+set -e
+DIR="$(dirname "$(readlink "$0")")"/..
+docker build -t docker-compose $DIR
+exec docker run -i -t -v /var/run/docker.sock:/var/run/docker.sock -v `pwd`:`pwd` -w `pwd` docker-compose $@
```

Comment:
Well, actually, there's a PR to change that, apparently, `/var/run` is deprecated and `/run/` is the new location; https://github.com/docker/docker/pull/10567